Userland/Applications/SoundPlayer/Player.cpp

```diff
@@ -38,7 +38,7 @@ Player::Player(Audio::ClientConnection& audio_client_connection)
     };
 }
 
-void Player::play_file_path(StringView path)
+void Player::play_file_path(const String& path)
```

Comment:
[The code style](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#const-placement) here is to use east-cost
```suggestion
void Player::play_file_path(String const& path)
```

Reply:
- [x] Use east-const